<?xml version="1.0"?>
<!--
Runs a Prosilica camera in streaming mode.

Change the ip_address to match your camera.
For help configuring your camera or determining its IP address, see
http://www.ros.org/wiki/prosilica_camera/Tutorials.
-->

<launch>

  <!-- The camera node -->
  <node ns="right" name="color_camera" pkg="prosilica_camera" type="prosilica_node" output="screen">
	<param name="ip_address" type="str" value="192.168.21.101"/>
	<rosparam command="load" file="$(find cob_camera_sensors)/ros/launch/$(env ROBOT)/right.yaml"/>

	<remap from="camera/image_raw" to="image_raw"/>
	<remap from="camera/camera_info" to="camera_info"/>
  </node>

  <node ns="right" name="set_camera_info" pkg="cob_camera_sensors" type="set_camera_info.py" output="screen">
	<rosparam command="load" file="$(find cob_camera_sensors)/ros/launch/$(env ROBOT)/right.yaml"/>
  </node>
  
  <node ns="right" pkg="image_proc" type="image_proc" respawn="false" name="prosilica_image_proc" />

</launch>
